tasks/install.yml

```diff
@@ -1,31 +1,48 @@
 ---
 
-- name: Ensure hostname is resolvable
+- name: Ensure Hostname is Resolvable
   become: yes
   lineinfile:
     dest: /etc/hosts
     line: "127.0.0.1 localhost {{ ansible_hostname }}"
     regexp: ^127\.0\.0\.1
+  when: ansible_virtualization_type != 'docker'
 
-- name: Install ssmtp packages
+- name: Install SSMTP
   become: yes
   apt:
-    name: "{{ item }}"
-  with_items:
-    - ssmtp
-
-- name: Add ssmtp group for restricted usage
-  become: yes
-  group:
     name: ssmtp
-    state: present
-  when: ssmtp.restrict
 
-- name: Restrict ssmtp usage
-  become : yes
+- name: "Restrict SSMTP Usage to User {{ sansible_ssmtp_user }}"
```

Comment:
Again I think we just need one of these and assert that both user and group are specified.